src/main/java/infinityworks/com/iwtechtest/model/Authority.java

```diff
@@ -1,4 +1,4 @@
-package com.infinityworks.techtest.model;
+package infinityworks.com.iwtechtest.model;
```

Comment:
Java package names should always be reverse domain format, e.g. com.infinityworks.....